### ISSUE TITLE
Config Problems with MVC 4 

### ISSUE THREAD

**Issue by stevozilik** [smsohan/MvcMailer#48]
Hi,

The MvcMailer NuGet package messes up the Web.config for Views in a MVC 4 beta Internet Application.

Steps to reproduce:
- create a default template MVC 4 Internet Application
- install MvcMailer: install-package MvcMailer
- update Scaffolding as per http://stackoverflow.com/questions/10241797/error-scaffolding-with-mvcmailer-in-mvc-4 : update-package T4Scaffolding (the other command would not even run for me Update-Package MvcScaffolding)
- restart VS
- the View's Web.config contains duplicate entries referencing Razor 3.0 libraries. Since there are references to Razor 4.0 libraries, the app stops working. 

Resolution:
remove the redundant references to 3.0 libraries

**Comment by Eonasdan** [comment]
this line needs to be removed 

```
<host factoryType="System.Web.Mvc.MvcWebRazorHostFactory, System.Web.Mvc, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35" />
```

and this

```
 </pages>
<pages validateRequest="false" pageParserFilterType="System.Web.Mvc.ViewTypeParserFilter, System.Web.Mvc, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35" pageBaseType="System.Web.Mvc.ViewPage, System.Web.Mvc, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35" userControlBaseType="System.Web.Mvc.ViewUserControl, System.Web.Mvc, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31BF3856AD364E35">
```

**Comment by stevozilik** [comment]
that's correct. However the issue still remains, this should not happen in first place.
